python/ray/experimental/sgd/pytorch/pytorch_runner.py

```diff
@@ -68,8 +73,8 @@ def setup(self):
             self.training_set,
             batch_size=self.batch_size,
             shuffle=True,
-            num_workers=2,
-            pin_memory=False)
+            num_workers=8,
```

Comment:
we'll have to somehow make this programmatically adjustable without changing source